src/app/cube/home/components/what-clark/what-clark.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="collections">
+    <div class="title-text">
+      <div class="title__icon">
+        <i class="fas fa-shield-check"></i>
+      </div>
+      Securing the nation through education
+    </div>
+    <div class="what__description">
+      {{ copy.WHAT_BLURB }}
+      <a routerLink="/about">Learn more</a>
+    </div>
+    <div class="what__actions">
+        <div class="about__link">
+            <div role=button class="play-button" (activate)="openModal()" aria-label="Play CLARK Video"><i class="fal fa-play-circle"></i><span class="button-text">Watch our video</span></div>
```

Comment:
Why is this a div acting like a button?

Reply:
I've learned that it's okay to have a div act as a button. The role property will take care of the screen reader for us which is actually really cool.